db/tests/dependents/test_dependents.py

```diff
@@ -27,38 +26,14 @@ def test_correct_dependents_amount_and_level(engine, library_tables_oids):
     publishers_dependents_graph = get_dependents_graph(library_tables_oids['Publishers'], engine, [])
 
     publishers_dependents = _get_object_dependents(publishers_dependents_graph, library_tables_oids['Publishers'])
-    publications_dependents = _get_object_dependents(publishers_dependents_graph, library_tables_oids['Publications'])
-    items_dependents = _get_object_dependents(publishers_dependents_graph, library_tables_oids['Items'])
```

Comment:
Why are these tests being removed?

Reply:
Before this change, the dependents graph was returning me the table along with the foreign key it's contained in, but the purpose of this PR is to change this behaviour because tables don't directly depend on other tables.
It also made the tests much easier. Not that much nesting